mlrun/api/utils/builder.py

```diff
@@ -624,6 +624,7 @@ def build_runtime(
         runtime.status.state = mlrun.common.schemas.FunctionState.ready
         return True
     if build.base_image:
+        # TODO: ml-models was removed in 1.5.0. remove it from here in 1.7.0.
```

Comment:
Can't we do something about it now? Or we need to keep it in order to make old mlrun versions based clients?

Reply:
the latter